pages/learn/studio/create/characters/controllers/joystick.md

```diff
@@ -0,0 +1,58 @@
+# Joystick Character Controller
+
+## Introduction
+
+**Joystick** character controllers are a type of character controller that allows the player to move a character using an on-screen joystick.
+
+<Note type="warning">
+Joystick is still in beta and may have some limitations or bugs. Please report any issues you encounter to the Highrise Studio team.
+</Note>
+
+## Adding a Joystick Character Controller
+
+Luckily, Highrise Studio comes with a built-in Joystick character controller that you can use to control your characters. Here's how you can add a Joystick character controller to your character:
+
+<Note type="warning">
+You need version "0.12.2" or later of Highrise Studio to use the Joystick character controller.
+</Note>
+
+1. **Right-click** in the project view and select `Create > Highrise > Scene Variant` to create a new scene variant.
+2. Rename the scene variant to something like "JoystickScenePrefab".
+3. In the **JoystickScenePrefab**, add a component called `JoystickCharacterController`
+4. Remove the default `PlayerCharacterController` component from the **JoystickScenePrefab**
+5. Customize the Joystick properties in the `JoystickCharacterController` component
+6. Click "Use this prefab" to set it as the default scene variant
+
+## Customizing the Joystick
+
+You can customize the Joystick character controller by adjusting the following properties in the `JoystickCharacterController` component:
+
+1. **Footstep Walk Sound**: The sound that plays when the character is walking.
+2. **Footstep Run Sound**: The sound that plays when the character is running.
+3. **External Input Action**: The input action that triggers the joystick movement.
```

Comment:
I would specify that this is for external devices like a keyboard and a gamepad, most often needed on desktop clients.